pkg/asset/installconfig/gcp/usertags.go

```diff
@@ -0,0 +1,298 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	rscmgr "cloud.google.com/go/resourcemanager/apiv3"
+	rscmgrpb "cloud.google.com/go/resourcemanager/apiv3/resourcemanagerpb"
+	"github.com/googleapis/gax-go/v2/apierror"
+	"github.com/pkg/errors"
```

Comment:
Let's not introduce this deprecated lib in new code. Use `fmt.Errorf` with the `%w` verb instead.